Cql/Cql.Comparers/KeyValuePairComparer.cs

```diff
@@ -30,9 +30,9 @@ internal class KeyValuePairComparer<TKey,TValue> : ICqlComparer, ICqlComparer<Ke
         KeyValuePair<TKey, TValue> x,
         KeyValuePair<TKey, TValue> y,
         string? precision = null) =>
-        Comparer<TKey>.Default.Compare(x.Key, y.Key) switch
+        cqlComparer.Compare(x.Key, y.Key, precision) switch
         {
-            0     => Comparer<TValue>.Default.Compare(x.Value, y.Value),
+            0     => cqlComparer.Compare(x.Value, y.Value, precision),
```

Comment:
Was this a bug, the fact that precision wasn't passed on to the compare?

Reply:
Not sure, but using the .NET Comparer couldn't take a precision, so I'm guessing that would've been a bug. It would' not make sense to use precision on KeyValuePair itself, but rather on the Value part